### PR TITLE
[SELS-TASK] Category Read Implementation

### PR DESCRIPTION
**Pull request by christanplaza**
[Commands]
php artisan db:seed 

[Pre-condition]
Perform the commands above to store a single record
Proceed to /categories to see feature

[Expected Output]
A page that displays an Unordered List with an entry of 'Seeded Project'

[Notes]
None

### CODE REVIEW

**Comment by jeremiahC** [comment]
Suggestion:
If you want to use dummy data and you want to store multiple data you can use the database seeder for running command that will store in database and factory for creating dummy data. it's more easier than just using tinker and creating object item by item

File: resources/views/layout.blade.php [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+    <!-- Fonts -->
+    <link rel="dns-prefetch" href="//fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css?family=Nunito" rel="stylesheet">
+
+    <!-- Styles -->
+    <link href="{{ asset('css/app.css') }}" rel="stylesheet">
+</head>
+<body>
+    <div id="app">
+        <nav class="navbar navbar-expand-md navbar-light bg-white shadow-sm">
```
Comment (jeremiahC) [inline resources/views/layout.blade.php]: Can you create a partial file for navbar, and just include it using `@include`. It's will be easier for us to trace components.

Comment (christanplaza) [reply resources/views/layout.blade.php]: Will do.

Comment (christanplaza) [reply resources/views/layout.blade.php]: Updated in d3f8f4853ae919b7ae06d0bd208a5c00095a2345

File: resources/views/navbar.blade.php [hunk 2]
```diff
@@ -0,0 +1,51 @@
+
+        <nav class="navbar navbar-expand-md navbar-light bg-white shadow-sm">
```
Comment (jeremiahC) [inline resources/views/navbar.blade.php]: please remove some extra indentions

Comment (christanplaza) [reply resources/views/navbar.blade.php]: Resolved in 09e5fe60d3de17d7bffc0db84fe9623b961bb6d6

**Review by jeremiahC — APPROVED** [review]

**Pull request merged**